docs/UserGuide.md

```diff
@@ -135,7 +129,7 @@ Deletes the specified volunteer from the database.
 Format: `/v del VOLUNTEER_INDEX`
 
 * Deletes the volunteer at the specified `VOLUNTEER_INDEX`.
-* The index refers to the number before the event's name in the displayed volunteer list.
+* The index refers to the number before the volunteer's name in the displayed volunteer list.
 * The index **must be a non-negative integer** 1, 2, 3, …​
```

Comment:
Index must be a positive integer.

My bad, this was edited by me, but I forgot to change it when we switched from id to index.